GLMakie/src/gl_backend.jl

```diff
@@ -43,17 +43,9 @@ function get_texture!(context, atlas::Makie.TextureAtlas)
         return atlas_texture_cache[(atlas, context)][1]
     else
         require_context(context)
-        tex = Texture(
-            context, atlas.data,
-            minfilter = :linear,
-            magfilter = :linear,
-            # TODO: Consider alternatives to using the builtin anisotropic
-            # samplers for signed distance fields; the anisotropic
-            # filtering should happen *after* the SDF thresholding, but
-            # with the builtin sampler it happens before.
-            anisotropic = 16f0,
-            mipmap = true
-        )
+        # anisotropic filtering sometimes creates artifacts with aspect/distortion
+        # corrected anti-aliasing radius, mipmap seems irrelevant
+        tex = Texture(context, atlas.data, minfilter = :linear, magfilter = :linear)
```

Comment:
With anisotropic filtering: (dots between rows of text)
![Screenshot 2025-04-17 172348](https://github.com/user-attachments/assets/45a4d00a-2143-43a8-8ec6-577d49c28747)

Without:
![Screenshot 2025-04-17 172356](https://github.com/user-attachments/assets/afedbc30-9628-4709-965e-ef76257ac4a7)

I think what this pr adds may act as anisotropic filtering itself, so it might be redundant now. In GLMakie I also see no difference at all between mipmap on and off. I don't see how mipmaps would help here either.